.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    ".": "0.11.0"
+    ".": "0.12.0"
```

Comment:
Nit: I know the file was already there, but could you include an empty line at the end, please?